src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -667,3 +667,33 @@ class InterfacesMIB(metaclass=MIBMeta, prefix='.1.3.6.1.2.1.2'):
     # FIXME Placeholder
     ifSpecific = \
         SubtreeMIBEntry('2.1.22', if_updater, ValueType.OBJECT_IDENTIFIER, lambda sub_id: ObjectIdentifier.null_oid())
+
+class sysNameUpdater(MIBUpdater):
+    def __init__(self):
+        super().__init__()
+        self.db_conn = mibs.init_db()
+
+    def update_data(self):
+        return
+
+    def get_sys_name(self):
+        """
+        Subclass update interface information
+        """
+        self.db_conn.connect(mibs.CONFIG_DB)
+
+        device_metadata = self.db_conn.get_all(self.db_conn.CONFIG_DB, "DEVICE_METADATA|localhost")
```

Comment:
>device_metadata [](start = 8, length = 15)

I notice `PhysicalTableMIBUpdater.reinit_data()` fetch `DEVICE_METADATA` periodically. Could you reuse that?
For device name changing case, I think the latency is good enough. No need to query Redis on each query.

Reply:
That data is coming from STATE_DB and not CONFIG_DB.

Reply:
Modified implementation to get hostname in reinit_data() function which gets invoked every minute.
Also, using hostname from system as default hostname during intialization.